db/migrate/20221123092506_create_likes.rb

```diff
@@ -1,11 +1,13 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
```

Comment:
There are some migration errors so database tables are not migrated. Kindly make sure that database has no errors and schema reflects the changes requested by previous code reviewer.